### PR TITLE
fix: terminal overflow on docs

### LINKED ISSUE #6025: [UI] Meshery docs terminal heading icons and copy button on mobile view move with horizontal scrolling
#### Current Behavior

Copy to clipboard button and bash heading text of terminal in meshery docs mobile view scrolls in horizontal scrolling
#### Desired Behavior

The buttons should be static.
#### Screenshots/Logs

meshery docs issue

#### Environment

- Browser: Chrome Safari Firefox
- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by adarsh500**
Signed-off-by: adarsh50 <adarshsulegai50@gmail.com>

**Description**

This PR fixes #6025 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

Previous behaviour:

https://user-images.githubusercontent.com/73278151/184551325-803c7601-be04-4449-9040-40bcf691a74a.mov

Updated behaviour: 

https://user-images.githubusercontent.com/73278151/184551347-cd1ca4e2-0109-4e6c-b5b4-1ec64bf93596.mov

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 899e9b217fbfb7ded12d72d5a0ba3fc6bdb1126f at: https://62f9428d6ae19325d6f6dbb5--meshery-docs-preview.netlify.app

**Comment by YashKamboj** [comment]
@sourav-sharma7 Please give your review on it

**Review by YashKamboj — DISMISSED** [review]
LGTM!! Great Work @adarsh500

**Comment by sourav-sharma7** [comment]
Looks good, Great work @adarsh500 🔥

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
Any chance we can have one scrollbar for the terminal instead of having a scroll for each line?
Not really great UX if there's a scrollbar at each line.

**Comment by l5io** [comment]
🚀 Preview for commit 22b3ea68d39d7b6207a0f7dc2beefbdb90809f11 at: https://62ff88db06d7aa01ff003f1e--meshery-docs-preview.netlify.app

**Comment by adarsh500** [comment]
I'm sorry for the delay, i was caught up with exams. Implementing this was tricky as the mac buttons and the bash text are css pseudo elements. But i believe i've got it to work @warunicorn19.

https://user-images.githubusercontent.com/73278151/185667613-76185ac8-8df0-4124-a6a9-1b20fb06c5de.mov

**Comment by l5io** [comment]
🚀 Preview for commit 643878827b3b55b57559d94437edc71415f7800a at: https://62ffbfdf6134770c05c38d21--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 1b3f9c7e1cf3278714429da318e25f1f3f916f8e at: https://62ffc26582d48a0de6ef6728--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 5644a4c072ecbb82bb19f9c55083d42100772201 at: https://62ffd680f0b2c91ef0040c5e--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 426b88e6bb16df4d96145f7f3ef5b87f8d562dc3 at: https://630324cd5f38793d2389217f--meshery-docs-preview.netlify.app

**Comment by adarsh500** [comment]
Hey @warunicorn19 I've made the changes you requested, could you please take a look at it?

File: docs/v0.5/assets/css/sass.css [hunk 1]
```diff
@@ -15,11 +15,11 @@
 
 pre.codeblock-pre { display: inline-flex; justify-content: left; text-align: left; align-items: left; align-content: left; flex-wrap: nowrap; flex-direction: row; background-color: #111; margin: 0; padding: 10px; border-radius: 0.3rem; width: 100%; }
 
-.codeblock { width: 100%; display: inline-flex; overflow: auto; justify-content: left; color: #ddd; margin: 0px; padding: 0px; }
+.codeblock { position: realative; width: 100%; display: inline-flex; overflow: auto; justify-content: left; color: #ddd; margin: 0px; padding: 0px; }
```
Comment (warunicorn19) [inline docs/v0.5/assets/css/sass.css:18]:
```suggestion
.codeblock { position: relative; width: 100%; display: inline-flex; overflow: auto; justify-content: left; color: #ddd; margin: 0px; padding: 0px; }
```

Comment (warunicorn19) [reply docs/v0.5/assets/css/sass.css:18]: Since there's a spelling mistake here, i wondering if this is a required change?

Comment (adarsh500) [reply docs/v0.5/assets/css/sass.css:18]: Nope that change wouldn't be necessary, adding/removing position: relative doesn't make an impact on the functionality. So we can just remove it entirely.

**Review by warunicorn19 — APPROVED** [review]
LGTM.Thank you @adarsh500

**Pull request merged**